routes/index.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
+const router = express.Router();
 
 const userRouter = require('./userRouter');
+const productRouter = require('./productRouter');
```

Comment:
```suggestion
const express = require('express');

const userRouter = require('./userRouter');
const productRouter = require('./productRouter');

const router = express.Router();
```
이전에도 동일한 내용 남겼던 기억이 있는데, 마찬가지로 코드 작성 순서 수정입니다.